main.go

```diff
@@ -575,6 +573,32 @@ func main() {
 		log.Info("Addon status reporting is not enabled")
 	}
 
+	if hubCfg != nil {
+		configFiles = append(configFiles, opts.hubConfigPath)
+
+		if hubCfg.TLSClientConfig.CertFile != "" {
+			configFiles = append(configFiles, hubCfg.TLSClientConfig.CertFile)
+		}
+	}
+
+	configChecker, err := addonutils.NewConfigChecker("config-policy-controller", configFiles...)
```

Comment:
To be on the safe side, if `len(configFiles) == 0`, could you please fall back to using `healthz.Ping`?